Runtime/Native/iOS/NativeClient.cs

```diff
@@ -55,20 +55,23 @@ internal struct Entry
 
 #endif
 
-        public NativeClient(string gameObjectName, BacktraceConfiguration configuration)
+        public NativeClient(BacktraceConfiguration configuration, List<string> attachments)

```

Comment:
Consider adding attachments to configuration object

Reply:
How can user define path to attachments in the Untiy IDE - because BacktraceConfiguration class contains a definition of everythign that user see and can edit in the UI. We can right now initialize it only via Initialize method so we can't also add attachments to backtraceConfiguration object because otherwise we user will somehow see them in the UI. For the same reason we're not able to define attributes in the UI.

Reply:
Hey! I didn't know the Config object was tightly coupled to the UI options - why does that need to be 1:1? Changing something in the UI will have a ripple effect (let's say we want to hide something in the UI and move it to code-only) and that seems like having a HTML field submit name be the same as the database field.

I just base this on the generic "prefer objects to parameters" especially if you already have an object you're passing (for example [here](https://www.codeguru.com/columns/dotnet/using-objects-as-parameters-in-c.htm)). Which is obviously subjective.

Meta: my suggestions are mainly as a passionate ex-developer and you can ignore them if you think they're hogwash :) Up to you, just trying to have add value with my grandpa developer skills :)